src/main/java/Duke.java

```diff
@@ -1,10 +1,163 @@
+                    System.out.println("input = " + input + "\n☹ OOPS!!! Please retype your input!");
+                    input = in.nextLine();
+                } catch (ArrayIndexOutOfBoundsException e2) {
+                    System.out.println("you have typed in = " + input + "\n☹ OOPS!!! where is your /by my friend? ");
+                    input = in.nextLine();
+                }
+
+            } else if (input.contains("event")) {
+                try {
+                    Task whatEvent = eventMethod(input);
+                    System.out.println("    Got it. I've added this task:\n" + "    " + whatEvent + "\n"
+                            + "    Now you have " + inputCount + " tasks in the list.");
+                    
+                    input = in.nextLine();
+                } catch (StringIndexOutOfBoundsException e1) {
+                    System.out.println("input = " + input + "\n☹ OOPS!!! Please retype your input!");
+                    input = in.nextLine();
+                } catch (ArrayIndexOutOfBoundsException e2) {
+                    System.out.println("you have typed in = " + input + "\nw☹ OOPS!!! where is your /at brother? ");
+                    input = in.nextLine();
+                }
+
+            } else {
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");//overall error check
+                input = in.nextLine();
+
+            }
+        }
+        System.out.println("    Bye! Hope to see you again soon!");
+
     }
+
+    private static String printList(Scanner in) {
+        String input;
+        System.out.println("    Here are the tasks in your list:");
+
+        for (int outputCount = 0; outputCount < inputCount; outputCount++) {
+            System.out.println("    " + (outputCount + 1) + "." + tasks[outputCount]);
+        }
+        input = in.nextLine();
+        return input;
+    }
+
+    private static String testTaskDone(String input, Scanner in) {
+        int taskNumber;
+        String[] inputSplitter;
+        inputSplitter = input.split(" ");
+        taskNumber = Integer.parseInt(inputSplitter[1]);
+        taskNumber = taskNumber - 1;
+
+        if (((taskNumber + 1) > 0) && ((taskNumber + 1) <= inputCount)) { //check for invalid inputs
+            input = taskDone(in, tasks[taskNumber]);
+
+        } else {
+            System.out.println("    Invalid input! Please check the list again!");
+            input = in.nextLine();
+        }
+        return input;
+    }
+
+    private static String taskDone(Scanner in, Task task) {
+        String input;
+        System.out.println("    Nice! I've marked this task as done:" + System.lineSeparator());
+        System.out.println("    " + task + " has been updated to -->");
+
+        task.markAsDone(); //mark x in [ ]
+
+        System.out.println("    " + task);
+        input = in.nextLine();
+        return input;
+    }
+
+    public static Task toDoMethod(String input) {
```

Comment:
Perhaps the method name to be reprhased so that it is more descriptive about what it does? Also, consider using JavaDoc comments for non-private classes / methods and non-trivial private methods in an appropriate way, as per https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/documentation.html#javadoc.